.github/workflows/rust.yml

```diff
@@ -85,3 +85,26 @@ jobs:
         with:
           command: build
           args: --release --verbose --all-features --target x86_64-unknown-linux-gnu
+  
+    build-container:
+    name: Build and Push image to DockerHub.
+    runs-on: ubuntu-latest
+    needs: test
+    if: github.ref == 'refs/heads/master'
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: true
+          tags: ayratbadykov/el_monitorro:latest
```

Comment:
@pxp9 can we call it `dev`?

Reply:
[comments](https://github.com/ayrat555/el_monitorro/pull/239/commits/9e477ec9d084f45cb6beb4c0146aa478afcdfbdd)